### PR TITLE
Add more Spanish aliases

### PR DESCRIPTION
**Pull request by epidemian**
First of all, thank you for this incredibly useful project!

Upon installing and using this config, i noticed that a couple of git commands i frequently use were missing Spanish aliases, and reverting to English was incredibly sad. This PR is an attempt at remedying that :relaxed: 

Some notes on the chosen aliases:
- I've never understood if `init` it means "in it" (which is totally reasonable for creating a repository where you're gonna "put things _in it_") or if it is an abbreviation of "initialize". So i added two aliases to cover both meanings (similar to `tronco = log`)
- I first considered using `relojear` for `checkout`, which i think would be the most appropriate translation for Argentinian users (e.g, "hey, were you _checking out_ my parner?" -> "che ¿estabas _relojeando_ a mi pareja?"). But it was most likely going to confuse other Hispanic people, so i left the more neutral `revisar`.
    
    Maybe we could have an `es-AR-gitconfig` file that includes the aliases from the base Spanish config and adds its own Argentinian slang words (e.g., `encanutar = stash`, `arrugar = revert`). What do you think?

Cheers!

### CODE REVIEW

**Comment by epidemian** [comment]
:recycle: I took the liberty of adding a couple more aliases to this PR :)

Also, i couldn't find a good Spanish translation for one of the most useful and non controversial of all git commands: `rebase`! :scream:

**Comment by lucasefe** [comment]
I'm sorry I missed this PR <3. I'll take a look at it soon.

File: spanish-gitconfig [hunk 1]
```diff
@@ -2,13 +2,21 @@
   agregar = add
   bitacora = log
   clonar = clone
+  control = remote
```
Comment (lucasefe) [inline spanish-gitconfig]: You got me here. Please, explain.

Comment (epidemian) [reply spanish-gitconfig]: Debatable but, for example, if the TV remote is missing, someone might ask "have you seen the remote?" or "dónde quedó el control?" in Spanish.

It's a very tongue-in-cheek alias. A more straightforward one would of course be `remoto`, or `lejano`/`distante`.

Comment (lucasefe) [reply spanish-gitconfig]: I'm down to add it to the soon to be available `AR-es` version, and use `distante` instead. I'll make the changes soon. 

This was great.

Comment (epidemian) [reply spanish-gitconfig]: Ok, i can change it to `distante` here. Is there anything else you'd like this PR to include? (I was lazy and did the first commit editing the file through the Github web interface)

**Comment by lucasefe** [comment]
That will be all!

**Review by lucasefe — APPROVED** [review]

**Comment by epidemian** [comment]
Thanks! 

This project is great. It feels like contributing to the future of international DVCS (?)

**Pull request merged**